multiversion/src/main/scala/multiversion/commands/LintCommand.scala

```diff
@@ -88,34 +86,22 @@ case class LintCommand(
               if tdep.dependency != Some(dep)
             } yield tdep
         }.toSet
-        val redundant = errors.foreach {
+
+        errors.foreach {
           case (module, versions) =>
             val deps = versions
               .map(v => SimpleDependency(module, v))
               .flatMap(index.byDependency.get(_))
             if (!deps.exists(isTransitive)) {
```

Comment:
@eed3si9n I don't understand this condition. Why is it fine to depend on multiple versions, as long as at least one of these versions is a transitive dependency?

Reply:
I am not sure if that's correct. The check we should perform is if the scala_import rules are evicted or not. If you have more than one non-evicted rules, that should correspond to multiple (e.g.) Guava JARs.